google/cloud/bigquery/opentelemetry_tracing.py

```diff
@@ -152,5 +152,11 @@ def _set_job_attributes(job_ref):
 
     if job_ref.num_child_jobs is not None:
         job_attributes["num_child_jobs"] = job_ref.num_child_jobs
+    
+    if job_ref.total_bytes_billed is not None:
+        job_attributes["total_bytes_billed"] = int(job_ref.total_bytes_billed)
```

Comment:
Since this property only applies query jobs (https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#jobstatistics2), but this code runs for all job types, let's protect this with [getattr](https://docs.python.org/3.12/library/functions.html#getattr) with a default.

Also, I don't think the conversion to `int` is required, right?

```suggestion
    total_bytes_billed = getattr(job_ref, "total_bytes_billed", None)
    if total_bytes_billed is not None:
        job_attributes["total_bytes_billed"] = total_bytes_billed
```